functions/Get-DbaServerAudit.ps1

```diff
@@ -42,43 +38,40 @@ Returns all Security Audits for the local and sql2016 SQL Server instances
 #>
 	[CmdletBinding()]
 	Param (
-		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
-		[DbaInstanceParameter]$SqlInstance,
+		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $true)]
+		[DbaInstanceParameter[]]$SqlInstance,
 		[PSCredential][System.Management.Automation.CredentialAttribute()]$Credential,
 		[switch]$Silent
 	)
 	
-	process
-	{
+	process {
 		foreach ($instance in $SqlInstance)
 		{
-			Write-Verbose "Attempting to connect to $instance"
-			try
-			{
+			try {
+				Write-Message -Level Verbose -Message "Connecting to $instance"
 				$server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $Credential
 			}
-			catch
-			{
+			catch {
 				Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
 			}
 			
-			if ($server.versionMajor -lt 10)
-			{
+			if ($server.versionMajor -lt 10) {
 				Write-Warning "Server Audits are only supported in SQL Server 2008 and above. Quitting."
 				continue
 			}
-			
-			foreach ($audit in $server.Audits)
-			{
+			foreach ($audit in $server.Audits) {
 				Add-Member -InputObject $audit -MemberType NoteProperty -Name ComputerName -value $audit.Parent.NetName
 				Add-Member -InputObject $audit -MemberType NoteProperty -Name InstanceName -value $audit.Parent.ServiceName
 				Add-Member -InputObject $audit -MemberType NoteProperty -Name SqlInstance -value $audit.Parent.DomainInstanceName
 				
 				Select-DefaultView -InputObject $audit -Property ComputerName, InstanceName, SqlInstance, Name, 'Enabled as IsEnabled', FilePath, FileName
 			}
+			if($server.Audits.Count -eq 0) {
+				Write-Message -Level Output -Message "No server audit found on $($server.DomainInstanceName)"
+			}
 		}
 	}
-	end { 
-	     Test-DbaDeprecation -DeprecatedOn "1.0.0" -Silent:$false -Alias Get-SqlServerAudit
+	end {
+		Test-DbaDeprecation -DeprecatedOn "1.0.0" -Silent:$false -Alias Get-SqlServerAudit
```

Comment:
@potatoqualitee this command needs to be added to the `dbatools.psm1` as it is not currently.

```powershell
if (-not (Test-Path Alias:Get-SqlServerAudit)) { Set-Alias -Scope Global -Name Get-SqlServerAudit -Value Get-DbaServerAudit }
```